web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -421,11 +425,12 @@
 
 
       //Update Links for social media
+      $scope.showSocialBarInFooter = gnGlobalSettings.gnCfg.mods.footer.showSocialBar;
       $scope.socialMediaLink = $location.absUrl();
       $scope.$on('$locationChangeSuccess', function(event) {
         $scope.socialMediaLink = $location.absUrl();
         $scope.showSocialMediaLink =
-            ($scope.socialMediaLink.indexOf('/metadata/') != -1);
+          (($scope.socialMediaLink.indexOf('/metadata/') == -1) && ($scope.showSocialBarInFooter));
```

Comment:
Is this for the social media links in the metadata detail page? If so, not very clear why is related to the `showSocialBarInFooter`

Reply:
No, it's for all pages. In the metadata page the social buttons were already hidden